charts/db-operator/templates/webhook/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.webhook.name }}
+  labels:
+    app: {{ include "webhook.name" . }}
+    app.kubernetes.io/name: {{ include "webhook.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: "webhook"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "webhook.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: "webhook"
+  template:
+    metadata:
+      labels:
+        app: {{ include "webhook.name" . }}
```

Comment:
Again duplicating labels, maybe we can get rid of the `app` one

Reply:
I deleted the `app` labels in https://github.com/db-operator/charts/pull/13/commits/0302ef6e403ca1ac01fa7a575d839c124ef4793b